analysis/blacklisted/blacklisted.go

```diff
@@ -42,11 +44,20 @@ type (
 	// UrlShort is a shortened version of the URL datatype that only accounts
 	// for the IP and url (hostname)
 	UrlShort struct {
-		Url string   `bson:"host"`
-		IPs []string `bson:"ips"`
+		Url string `bson:"host"`
```

Comment:
The ips found from the urls/hostnames table were duplicates of those found in hosts.